resources/dicts/patrols/general/training.json

```diff
@@ -9673,5 +9673,264 @@
+                       "reg_death": "m_c was killed after training in the Dark Forest."
+                }
+            }
+        ]
+    },
+    {
+        "patrol_id": "gen_train_darkforest2",
+        "biome": ["Any"],
+        "season": ["Any"],
+        "types": ["training"],
+        "tags": [],
+        "patrol_art": "gen_train_darkforest",
+        "min_cats": 2,
+        "max_cats": 2,
+        "min_max_status": {
+            "all apprentices": [1, 1],
+            "normal adult": [1, 1]
+        },
+        "pl_skill_constraint": ["DARK,2"],
+        "weight": 10,
+        "intro_text": "r_c, right on p_l's heels, complains that {PRONOUN/r_c/subject} {VERB/r_c/don't/doesn't} want to do more basic training. p_l insists that this training session, however, would be vastly different. As they arrive to their sparring area, a ragged, misty figure bearing dry scars waits in the area.",
+        "decline_text": "Shaken by the look of the cat, r_c turns away and pelts off back to camp, despite p_l's protests.",
+        "chance_of_success": 50,
+        "success_outcomes": [
+            {
+                "text": "p_l gives an encouraging nod to r_c as the spirit introduces themself and prepares to train with r_c. r_c hesitantly trains with the spirit under p_l's supervision, and {PRONOUN/r_c/subject} {VERB/r_c/come/comes} home with wounds. p_l insists that the training session was an introduction into how real warriors fight.",
+                "exp": 10,
+                "weight": 5,
+                "injury": [
+                    {
+                        "cats": ["r_c"],
+                        "injuries": ["claw-wound", "cat bite"],
+                        "scars": ["TWO", "SIDE", "THROAT"]
+                    }
+                ],
+                "history_text": {
+                    "reg_death": "m_c was killed after training in the Dark Forest.",
+                    "scar": "m_c was scarred after a brutal training session in the Dark Forest."
+                },
+                "relationships": [
+                    {
+                        "cats_to": ["patrol"],
+                        "cats_from": ["patrol"],
+                        "mutual": false,
+                        "values": ["platonic", "respect"],
+                        "amount": 5
+                    }
+                ]
+            },
+            {
+                "text": "This cat, this <i>spirit</i>, has a commanding presence. s_c softly encourages the apprentice to the battle, but also makes sure that r_c doesn't get too hurt by the spirit. The Dark Forest spirit argues that r_c would never find out how real warriors fight, but p_l retorts that this is r_c's first time, and they can focus on more battle moves later.",
+                "exp": 10,
+                "weight": 20,
+                "injury": [
+                    {
+                        "cats": ["r_c"],
+                        "injuries": ["torn pelt"],
```

Comment:
If you take my 'nothing marks app1's coat' suggestion above, I'd change this to sore and add a bruises injury and maybe a nice spain injury?